src/main/java/com/exalttech/trex/ui/views/statistics/StatsTableGenerator.java

```diff
@@ -436,7 +436,7 @@ private void addTotalColumn(double columnWidth, int columnIndex, double tx_bps_l
             if ((row.getKey() + "-total").startsWith("m_total_tx_bps_l1-total")) {
                 cell.updateItem("" + tx_bps_l1_total, "" + tx_bps_l1_total);
             }
-            if ((row.getKey() + "-total").startsWith("m_total_rx_bps_l1-total")) {
+            else if ((row.getKey() + "-total").startsWith("m_total_rx_bps_l1-total")) {
                 cell.updateItem("" + rx_bps_l1_total, "" + rx_bps_l1_total);
```

Comment:
Commit fd3cc81
File "StatsTableGenerator.java"
Lines 436 - 540
"Else" block is called if calculating Tx L1 total, so the data is overwriting.